aprslib/packets/telemetry.py

```diff
@@ -0,0 +1,32 @@
+from aprslib.packets.base import APRSPacket
+
+class TelemetryReport(APRSPacket):
+    format = 'raw'
+    sequenceno = 0
+    analog1 = 0
+    analog2 = 0
+    analog3 = 0
+    analog4 = 0
+    analog5 = 0
+    digitalvalue = ['0']*8
```

Comment:
Just keep the value as a `int` as is easier to manipulate. Then for serialization do `'{:b}'.format(self.digitalvalue)`